vm/README.md

```diff
@@ -43,6 +43,32 @@ In our CI, we build it using:
 - `binutils`: `2.23.2`
 - `llvm`: `3.8` on linux; `4.0.1` on darwin
 
+---
+**NOTE:**
+
+On certain versions of MacOS you might run into build errors of the form:
```

Comment:
Please specify which versions. This seems specific to 10.15 (Catalina).

Reply:
@dougxc, thank you for the review. I've updated the PR with the change.